### PR TITLE
fix merge logic

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/versioning/BlobDAORdbImpl.java [hunk 1]
```diff
@@ -716,26 +718,25 @@ private String getStringFromLocationList(List<String> locationList) {
...
+        blobContainerList.add(BlobContainer.create(blobExpandedEntry.getValue()));
       }
-      locationBlobsMap.putAll(locationBlobsMapNew);
-      List<BlobContainer> blobContainerList = new LinkedList<>();
-      for (Map.Entry<String, BlobExpanded> blobExpandedEntry : locationBlobsMap.entrySet()) {
-        if (blobExpandedEntry.getValue() != null) {
-          blobContainerList.add(BlobContainer.create(blobExpandedEntry.getValue()));
-        }
-       }
+    }
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/versioning/BlobDAORdbImpl.java:739]: formatting

File: backend/src/main/java/ai/verta/modeldb/versioning/BlobDAORdbImpl.java [hunk 2]
```diff
@@ -602,7 +603,8 @@ private CommitEntity getCommonParent(Session session, String commitA, String com
                   LinkedHashSet<BlobExpanded> newHash = new LinkedHashSet<>(m1);
                   newHash.addAll(m2);
                   return newHash;
-                }, LinkedHashMap::new));
+                },
+                LinkedHashMap::new));
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/versioning/BlobDAORdbImpl.java:607]: auto formatting

**Comment by ravishetye** [comment]
I resolved the issue by breaking the session into read and write. So this PR is not required. However if there are any other changes which we should not throw away please file a separate PR.

**Review by ravishetye — CHANGES_REQUESTED** [review]
not needed

**Pull request closed without merging**